CodeHub/Services/NavigationService.cs

```diff
@@ -106,5 +130,73 @@ public NavigationService(CustomFrame frame)
             NavigationSemaphore.Release();
             return result;
         }
+
+        /// <summary>
+        /// Search for the Page Title with the given Menu type
+        /// </summary>
+        /// <param name="type">type of the Menu</param>
+        /// <returns>string</returns>
+        /// <exception cref="Exception">When the given type don't have a Page Title pair</exception> 
+        public string ChoosePageTitleByPageType(Type type)
```

Comment:
It would be great if you could make this switch case instead of if else 👍

Reply:
I totally agree with You, but I can't because if I want to use this code:
```` cs
switch (type)
{
    case typeof(CommentView): return "Comment";
}
````
... get the following error:
`A constant value is expected.`

_[SO](https://stackoverflow.com/a/7542810/7306734) answer._

Maybe we can find some workaround but maybe this is not the most important now. :) But I agree with You :)

Reply:
Yeah that's fine by me too. If you don't have more commits to make, I can go ahead and merge this.

Reply:
Not yet pleay, I need more commits because some problems appeared.

Reply:
ok sure 👍